cylc/flow/remote.py

```diff
@@ -180,9 +180,14 @@ def get_includes_to_rsync(rsync_includes=None):
 DEFAULT_RSYNC_OPTS = [
     '-a',
     '--checksum',
-    '--out-format=%o %n%L',
+    '--out-format=%i %o %n%L',  # see comment below
```

Comment:
Do we want to include `%i` in the logs?

* If this is only really relevant to the dry-run functionality, then continue to override the CLI argument as we did before.
* If we think we should have this in the logs, then to keep them readable it might be better to keep the operation at the start of the line?

Reply:
The rsync output isn't included in the reinstall logs. The only thing we do with the rsync output is print it to stdout in the dry-run case.

Reply:
I think it is, install a workflow, make changes, then check `~/cylc-run/<workflow>/log/install/*`.

Reply:
Ah ok - it's only included in the `01-install.log`. So I should make this option with `%i` only apply to reinstall

Reply:
Will tackle in follow-up.